gpg-client-wrapper

```diff
@@ -5,10 +5,56 @@ eoo=0       # end of options reached
 output=0    # do we try to write to file
 target=()   # where do we try to write to
 fd_for_stdout=63 # some (hopefuly) unused FD, later redirected to stdout
+localgpg=0 #use local gpg (for ex --gen-rand etc.)
+origargs="$@"
```

Comment:
Use array, otherwise options with spaces will be broken.
`origargs=( "$@" )`

Reply:
fixed it